x/gamm/keeper/pool_service_test.go

```diff
@@ -285,7 +285,7 @@ func (suite *KeeperTestSuite) TestInitializePool() {
 			createPool: func() poolmanagertypes.PoolI {
 				return suite.PrepareConcentratedPool()
 			},
-			expectPanic: true,
+			expectPass: false,
```

Comment:
Why do we no longer run expectPanic? If this is intended, can we remove it as a param since this is the only test case that used it?

Reply:
The panic was changed to error because I added an error check in the method for other implementation needs. So instead of panicking before, now it errors out

Reply:
done